src/extlibs/hrvo/hrvo_agent.cpp

```diff
@@ -444,23 +442,31 @@ void HRVOAgent::computePreferredVelocity()
         return;
     }
 
-    std::unique_ptr<Goal> &nextGoal = simulator_->goals_[goal_index_];
+    std::unique_ptr<Goal> &nextGoal = simulator_->goals[goal_index_];
     Vector2 goalPosition            = nextGoal->getCurrentGoalPosition();
     float speedAtGoal               = nextGoal->getDesiredSpeedAtCurrentGoal();
     Vector2 distVectorToGoal        = goalPosition - position_;
-    auto distToGoal = static_cast<float>(std::sqrt(std::pow(distVectorToGoal.getX(), 2) +
-                                                   std::pow(distVectorToGoal.getY(), 2)));
+    auto distToGoal                 = abs(distVectorToGoal);
+    // Increasing deceleration distance to reduce the chance of overshooting the
+    // destination
+    float decel_dist_multiplier = 1.2f;
```

Comment:
can we push this as constants in the class so it's easier to tune

Reply:
Moved to class